extensions/mssql/package.json

```diff
@@ -774,6 +783,11 @@
           "command": "mssql.dropObject",
           "when": "connectionProvider == MSSQL && nodeType =~ /^(ServerLevelLogin|User|ServerLevelServerRole|ApplicationRole|DatabaseRole)$/ && !(nodePath =~ /^.*\\/System Databases\\/.*$/)  && config.workbench.enablePreviewFeatures",
           "group": "inline@3"
+        },
+        {
+          "command": "mssql.restoreDatabase",
+          "when": "connectionProvider == MSSQL && nodeType && nodeType == Database && !(nodePath =~ /^.*\\/System Databases\\/.*$/) && config.workbench.enablePreviewFeatures",
```

Comment:
Do we need to skip cloud DBs here?

Reply:
yes! I have added in other commit, but still need lot of changes.